### PR TITLE
fix: resolve the realpath of plugin path

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)

loadPlugin

##### Description of change

如果插件路径是软链接，将会转成真实链接

### CODE REVIEW

**Comment by popomore** [comment]
过了

File: test/loader/mixin/load_plugin.test.js [hunk 1]
```diff
@@ -370,4 +372,15 @@ describe('test/load_plugin.test.js', function() {
     loader.plugins.hsfclient.path
       .should.equal(utils.getFilepath('plugin-path-package/node_modules/hsfclient'));
   });
+
+  it('should resolve the realpath of plugin path', () => {
+    rimraf.sync(utils.getFilepath('realpath/node_modules/a'));
+    fs.symlinkSync('../a', utils.getFilepath('realpath/node_modules/a'), 'dir');
```
Comment (popomore) [inline test/loader/mixin/load_plugin.test.js]: 需要动态创建，兼容 Windows

**Comment by atian25** [comment]
+1

**Comment by popomore** [comment]
合了，发个 minor

**Comment by popomore** [comment]
+ egg-core@0.2.1

**Pull request merged**